### ISSUE TITLE
move additional stop info to a device

### ISSUE THREAD

**Issue by behrisch** [eclipse/sumo#7702]
Some information such as arrival and depart time are only stored in the stop for retrieval via traci. They could be merged with the stopinfo output and moved to a separate device. This would make the code cleaner because it separates data collection from values needed for processing the stop. Furthermore state saving could be delegated to the device.

**Comment by namdre** [comment]
Yes. Though 'arrival' is the declared expected arrival (used for computing delay and writing output) whereas actualArrival is collected data.
